lib/crypto/tasks.py

```diff
@@ -24,43 +24,49 @@ def __exit__(self, type, value, traceback):
 
 
 @task
-def sign_addons(addon_ids, **kw):
+def sign_addons(addon_ids, force=False, **kw):
     log.info('[{0}] Signing addons.'.format(addon_ids))
     for version in Version.objects.filter(
             addon_id__in=addon_ids, addon__type=amo.ADDON_EXTENSION):
         log.info('Signing addon {0}, version {1}'.format(version.addon,
                                                          version))
-        # We need to bump the version number of the file and the version, so
+        # We need to bump the version number of the file and the Version, so
         # the Firefox extension update mecanism picks this new signed version
         # and installs it.
-        bump_version_number(version)
+        if force:
+            to_sign = version.files.all()
+        else:
+            to_sign = [f for f in version.files.all() if not f.is_signed]
+        if not to_sign:
```

Comment:
I was hoping to log which files we weren't signing in the output so we could easily tell what happened.  What do you think?